resources/default_options.js

```diff
@@ -37,5 +38,6 @@ module.exports = {
 	protocol: EConnectionProtocol.Auto,
 	language: 'english',
 	webCompatibilityMode: false,
-	saveAppTickets: true
+	saveAppTickets: true,
+	maxTimeout: null
```

Comment:
I first implemented it using 10000 as default value. But the timeouts vary quite a lot throughout the code. Therefore I decided to rather implement it by using null so that the timeout values remain at the value prior this change.